visualization/app/codeCharta/ui/matchingFilesCounter/matchingFilesCounter.component.ts

```diff
@@ -0,0 +1,61 @@
+import "./matchingFilesCounter.component.scss"
+import { BlacklistType, BlacklistItem, CodeMapNode, Settings, RecursivePartial } from "../../codeCharta.model";
+import { SettingsServiceSubscriber, SettingsService } from "../../state/settings.service";
+import { CodeMapHelper } from "../../util/codeMapHelper";
+import { IRootScopeService, IAngularEvent } from "angular";
+import { NodeSearchService, NodeSearchSubscriber } from "../../state/nodeSearch.service";
+
+export class MatchingFilesCounterController implements NodeSearchSubscriber, SettingsServiceSubscriber {
+
+    private _viewModel: {
+		fileCount: number
+		hideCount: number
+		excludeCount: number
+		searchPattern: string
+		blacklist: BlacklistItem[]
+	} = {
+		fileCount: 0,
+		hideCount: 0,
+		excludeCount: 0,
+		searchPattern: "",
+		blacklist: []
+	}
+
+	private searchedNodeLeaves: CodeMapNode[] = [] // TODO: geht das in die DynamicSettings oder service
```

Comment:
TODO:
This `private searchedNodeLeaves` stays there and as it is not updated here, we don't need to notify/call anybody else. or did I miss something?

Reply:
the todo is a leftover and outdated, i will remove it.